dev-qt/libqt4pas/libqt4pas-2.5.0.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="Pascal Qt interface"
+DESCRIPTION="Pascal wrapper around Qt interfaces.
+Just the necessary stuff for Lazarus is wrapped here."
+HOMEPAGE="http://users.telenet.be/Jan.Van.hijfte/qtforfpc/fpcqt4.html"
+COPYRIGHT="2005-2011 Jan Van hijfte"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://users.telenet.be/Jan.Van.hijfte/qtforfpc/V2.5/qt4pas-V2.5_Qt4.5.3.tar.gz"
+CHECKSUM_SHA256="825423db80da4df5c21816c0392b3394cddfe2f3293dfd08ace84941726affea"
+SOURCE_DIR="qt4pas-V2.5_Qt4.5.3"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libqt4pas$secondaryArchSuffix = $portVersion
+	lib:libqt4pas$secondaryArchSuffix = 5.2.5 compat >= 5.2
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	lib:libqtcore$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libqt4pas$secondaryArchSuffix = $portVersion
+	#lib:libQt4Pas$secondaryArchSuffix = $portVersion compat >= 2.2
+	devel:libqt4pas$secondaryArchSuffix = 5.2.5 compat >= 5.2
+	"
+REQUIRES_devel="
+	haiku$secondaryArchSuffix
+	lib:libqtcore${secondaryArchSuffix}
+	devel:libqtcore${secondaryArchSuffix}
+	cmd:qmake$secondaryArchSuffix
+	cmd:qmake
+	cmd:make
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:make
+	cmd:qmake$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	lib:libqtcore$secondaryArchSuffix
+	devel:libqtcore$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc${secondaryArchSuffix}
+	cmd:ld${secondaryArchSuffix}
+	lib:libqtcore$secondaryArchSuffix
+	devel:libqtcore$secondaryArchSuffix
```

Comment:
Thanks for your latest fixes. You're almost done ;-)
You forgot, however, a couple of things:

* **`lib:libqtcore$secondaryArchSuffix`** and **`devel:libqtcore$secondaryArchSuffix`** should be removed from BUILD_PREREQUIRES because we don't need them here.

* As explained in the comment above, the following 3 **`cmd:`** lines, currently in BUILD_REQUIRES, should be moved to BUILD_PREREQUIRES:
```
	cmd:make
	cmd:qmake$secondaryArchSuffix
	cmd:g++$secondaryArchSuffix
```
BTW, if you could sort alphabetically the **`cmd:`** lines in BUILD_PREREQUIRES it would be perfect!

* While at it, could you please also drop, in the whole recipe, a few curly braces where this is possible?
To do so, each occurence of «**`${secondaryArchSuffix}`**», when **_not_** followed by «**`_devel`**», should be replaced by «**`$secondaryArchSuffix`**».
In other words, we shouldn't have any «**`${secondaryArchSuffix}`**» but only «**`$secondaryArchSuffix`**» and «**`${secondaryArchSuffix}_devel`**».